zebra/rt_netlink.c

```diff
@@ -2289,6 +2289,9 @@ ssize_t netlink_route_multipath_msg_encode(int cmd, struct zebra_dplane_ctx *ctx
 
 	/* Table corresponding to this route. */
 	table_id = dplane_ctx_get_table(ctx);
+	if (fpm)
+		table_id = dplane_ctx_get_vrf(ctx);
```

Comment:
Why are we overriding the table ID with the VRF ID here? Isn't this a breaking change?

Reply:
This part uses table_id as VRF lookup in fpmsyncd. This should be placed in the FRR patch of SONiC instead of being modified directly here. I have fixed it.